src/units/stimuli.sequence.dragdrop.pas

```diff
@@ -0,0 +1,208 @@
+{
+  Stimulus Control
+  Copyright (C) 2014-2020 Carlos Rafael Fernandes Picanço, Universidade Federal do Pará.
+  The present file is distributed under the terms of the GNU General Public License (GPL v3.0).
+  You should have received a copy of the GNU General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+}
+unit Stimuli.Sequence.DragDrop;
+
+{$mode objfpc}{$H+}
+
+interface
+
+uses
+  Classes, SysUtils, Controls, ExtCtrls, Schedules, Graphics
+  , Stimuli
+  , Stimuli.Abstract
+  , Stimuli.Image.DragDropable
+  , Stimuli.Image.Animation
+  , Stimuli.Image.Base
+  , Experiments.Grids
+  , Forms.Main
+  ;
+
+type
+
+  { TDragDropStimuli }
+
+  TDragDropStimuli = class(TStimulus, IStimuli)
+  private
+    LComparison : TDragDropableItem;
+    LItem : TDragDropableItem;
+    FSample : TDragDropableItem;
+    FAnimation : TAnimation;
+  public
+    constructor Create(AOwner : TComponent); override;
+    procedure ResetGrid;
+    function AsInterface : IStimuli;
+    procedure DoExpectedResponse;
+    procedure LoadFromParameters(AParameters : TStringList);
+    procedure Start;
+    procedure Stop;
+  end;
+
+implementation
+
+{ TDragDropStimuli }
+
+function TDragDropStimuli.AsInterface : IStimuli;
+begin
+  Result := Self;
+end;
+
+procedure TDragDropStimuli.DoExpectedResponse;
+begin
+
+end;
+
+procedure TDragDropStimuli.LoadFromParameters(AParameters: TStringList);
+begin
+
+end;
```

Comment:
O método `LoadFromParameters` (carregar a partir de parâmetros) serve para carregar as configuração do arquivo de configuração copiadas para o argumento `AParameters`. Esse argumento é um lista de strings que deverá estar no formato `INI`.

Reply:
Quais seriam essas configurações?

Reply:
Nós criamos uma unidade com a lógica do delineamento de cada experimento. Essa unidade usa essa lógica para escrever um arquivo de configuração. Esse arquivo é lido, tentativa a tentativa, quando executamos o início da sessão. A variável `AParameters` recebe justamente os parâmetros da tentativa atual. Por sua vez, o método `LoadFromParameters` lê apenas os parâmetros de interesse da classe.